plugins/flytekit-bigquery/tests/test_bigquery.py

```diff
@@ -0,0 +1,77 @@
+from collections import OrderedDict
+
+import pytest
+from flytekitplugins.bigquery import BigQueryConfig, BigQueryTask
+from google.cloud.bigquery import QueryJobConfig
+from google.protobuf import json_format
+from google.protobuf.struct_pb2 import Struct
+
+from flytekit import kwtypes, workflow
+from flytekit.extend import Image, ImageConfig, SerializationSettings, get_serializable
+from flytekit.types.schema import FlyteSchema
+
+query_template = """
+            insert overwrite directory '{{ .rawOutputDataPrefix }}' stored as parquet
```

Comment:
The way stuff is written today, this will be `gs://`.   Is this sufficient?  You weren't looking for a `bq://` prefix right?

Reply:
This `query_template` is copied from Athena, we won't do it in the real world. 
will change to another one.